csrc/ir/internal_nodes.h

```diff
@@ -1359,16 +1359,59 @@ class GroupedWelfordOp : public Expr {
 class NVF_API MmaOp : public Expr {
  public:
   using AxesData = std::vector<int64_t>;
+  // AxisMapping denotes the pairing of two input dimensions to produce an
+  // output dimension. It holds two vectors of integers indicating the
+  // corresponding position of each output axis in either the A or B input.
+  // Positions are absolute and refer to the noReductions logical domain. NOTE:
```

Comment:
`noReductions logical` does this mean we are ignoring K in the list?

Reply:
From the implementation, it doesn't seem to be ignoring K.

Reply:
Confusingly worded. It means reductions are ignored on the _inputs_ in case a reduction output is used in an MmaOp.